apps/web/ee/pages/api/integrations/stripepayment/webhook.ts

```diff
@@ -187,10 +187,6 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
     const event = stripe.webhooks.constructEvent(payload, sig, process.env.STRIPE_WEBHOOK_SECRET);
 
-    if (event.account) {
-      throw new HttpCode({ statusCode: 202, message: "Incoming connected account" });
-    }
```

Comment:
This was throwing 202 on my end and that body is received with an event.account value.